src/NovelRT/Windowing/WindowingService.cpp

```diff
@@ -113,6 +115,20 @@ namespace NovelRT::Windowing {
 
 
   }
+
+  void WindowingService::checkForOptimus() {
```

Comment:
nit: This should probably be called `tryLoadOptimus`. It would also be nice to not "leak" the handle.